doc/RecordingControl.xml

```diff
@@ -1684,8 +1766,212 @@ SetRecordingJobMode(JobToken, Active)
+        The <literal>MovieFragmentRandomAccessBox</literal> shall provide references for all tracks and their applicable <literal>MovieFragmentBox</literal> instances.
+      </para>
+      <para>
+        The device shall upload a span end object.
+        It should generate the object key for a span object with the <literal>"mp4_end"</literal> as the <literal>extension</literal>.
+      </para>
+    </section>
+    <section>
+      <title>Recording format CMAF</title>
+      <para>
+        For the recording format <literal>CMAF</literal>, the recording information shall be written as as one track per segment.
+        Each segment either contains one track with video data, one track with audio data, or one track with metadata, indicated by the extension.
+        When closing multiple types of segments of an underlying span at the same time, the device shall only increment the segment counter by 1.
+        The device shall generate the object key for each segment with the <literal>extension</literal> based on the type of data contained in the segment:
+        <itemizedlist>
+          <listitem>
+            <para>Video data: <literal>"m4v"</literal></para>
+          </listitem>
+          <listitem>
+            <para>Audio data: <literal>"m4a"</literal></para>
+          </listitem>
+          <listitem>
+            <para>Metadata: <literal>"m4m"</literal></para>
+          </listitem>
+        </itemizedlist>
+      </para>
+      <para>
+        The device shall upload each segment object with the content type based on the type of data contained in the segment:
+        <itemizedlist>
+          <listitem>
+            <para>Video data: <literal>video/mp4</literal></para>
+          </listitem>
+          <listitem>
+            <para>Audio data: <literal>audio/mp4</literal></para>
+          </listitem>
+          <listitem>
+            <para>Metadata: <literal>application/mp4</literal></para>
+          </listitem>
+        </itemizedlist>
+        The device should include the <literal>codecs</literal> parameter in the content type according to RFC 6381.
+      </para>
+      <para>
+        The device shall generate each segment according to ISO/IEC 23000-19, the common media application format (CMAF) for segmented media.
+      </para>
+      <para>
+        The device shall upload a span end object for each segment type.
+        It should generate the object key for each span end object with the <literal>extension</literal> based on the type of data contained in the segments:
+        <itemizedlist>
+          <listitem>
+            <para>Video data: <literal>"m4v_end"</literal></para>
+          </listitem>
+          <listitem>
+            <para>Audio data: <literal>"m4a_end"</literal></para>
+          </listitem>
+          <listitem>
+            <para>Metadata: <literal>"m4m_end"</literal></para>
+          </listitem>
+        </itemizedlist>
+      </para>
+    </section>
```

Comment:
The following attributes have been proposed:

| Key | Value | Format |
| -- | -- | -- |
| `Version` | 1.0 in initial version | `<major>.<minor>` |
| `Source` | Identifier of the physical device | String |
| `EventTopic` | Topic of the event that triggered the recording job | String |
| `Duration` | Duration of the segment | ISO 8601 |
| `VideoChannel` | Video channel identifier | String |
| `VideoBitrate` | Average video bitrate in kilobits per second | Integer |
| `VideoWidth` | Video width in pixels | Integer |
| `VideoHeight` | Video height in pixels | Integer |
| `VideoFramerate` | Average video framerate in frames per second | Float |
| `AudioChannel` | Audio channel identifier | String |
| `AudioBitrate` | Audio bitrate in kilobits per second | Integer |
| `AudioSampleRate` | Audio sample rate in kilohertz | Integer |

Reply:
This seems very useful to playback the files when the recording config or the device is not accessible.
Should VideoWidth and VideoHeight include cropping?

Reply:
To be useful, the values should match the information in the `MovieBox`, I would say.

Reply:
If you mean the VisualSampleEntry then yes cropping is included

Reply:
I added the list of attributes to the spec and will add more information regarding their use.